django/thunderstore/account/forms.py

```diff
@@ -25,7 +25,7 @@ def clean_team(self) -> Team:
     def save(self) -> ServiceAccount:
         owner = self.cleaned_data["team"]
         nickname = self.cleaned_data["nickname"]
-        (service_account, token) = ServiceAccount.create(owner, nickname)
+        (service_account, token) = ServiceAccount.create(owner, nickname, self.user)
```

Comment:
Could swap to using explicitly named arguments at this point already for clarity